### PR TITLE
To-Do-List (Feature/to do list structure)

### PR DESCRIPTION
**Pull request by lucy-sees**
In this project, I implemented the following:

- I set up the project with Webpack.
- I used HTML Webpack Plugin.
- I set an array of simple to-do tasks.
- I wrote a function to iterate over the tasks array and generate elements dynamically.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "checklister",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (LionRouge1) [inline package.json]: - [ ] Nice work configuring Webpack command for production mode (`"build": "webpack"`) :clap: However the command for building Webpack in development mode is not configured (`"start": "webpack serve --open"`). Kindly make sure it's configured and the command `npm run start` or `npm run dev` run the **dev server**.

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        exclude: /node_modules/, // Exclude node_modules
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
```
Comment (LionRouge1) [inline webpack.config.js]: - [ ] Nice work using Webpack for the implementation of the project :+1:  However Webpack **dev server** is not set up in your Webpack configuration. Kindly make sure Webpack **dev server** is set up and running with no errors

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    const li = document.createElement('li');
+    const check = document.createElement('input');
+    const lbl = document.createElement('label');
+    check.type = 'checkbox';
+    check.name = tasksArr[i].index.toString();
+    ul.append(li);
+    li.append(lbl);
+    lbl.append(check);
+    lbl.innerHTML += tasksArr[i].description;
+  }
```
Comment (LionRouge1) [inline src/index.js]: - [ ] Nice work implementing the list of tasks :clap: However your design is missing the left icon that help to remove a task which is located on the right side of each task
### Your design :point_down: 
Screenshot from 2023-08-17 20-24-23
### What it is supposed to look like :point_down: 
Screenshot from 2023-08-17 20-23-52

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @lucy-sees, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Nice work using Webpack :+1: 
- Good job storing tasks in a single JavaScript object :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [ ] The link to the live demo of the application is not working as it intend to. And since the deployment is to required for now, I suggest you remove the **Live Demo** section or make sure the link is actually linking to the live version of the application.That will make your project look more professional and clean.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @lucy-sees 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**